yarn.lock

```diff
@@ -6802,6 +6817,18 @@ sol-explore@^1.6.2:
   version "1.6.2"
   resolved "https://registry.yarnpkg.com/sol-explore/-/sol-explore-1.6.2.tgz#43ae8c419fd3ac056a05f8a9d1fb1022cd41ecc2"
 
+solc@0.5.0:
```

Comment:
I assume we need this because truffle 5.0 depends on it? Even though we aren't using it?

Reply:
Migrating requires adding bn.js for big numbers.  I'm updating the yarn.lock file.